generators/app/index.js

```diff
@@ -8,6 +8,10 @@ var banner = require('./banner')
 */
 module.exports = class extends Generator {
 
+  installingLodash() {
+    this.npmInstall(['wait-on'], { 'dev': true });
```

Comment:
Why are you installing this module in that way and why it's installed in 'dev' mode?

Reply:
because wait-on is a nice toll and I don't want to implement the waiting routine on my one. In our ACA / ADW projects it was always in the devdependencies scope. I hope with 'dev' it goes there too.

Reply:
I guess you may use `save-dev` parameter.
https://yeoman.io/authoring/dependencies.html

Reply:
Thanks Angel :)